pkg/agent/ecs.go

```diff
@@ -0,0 +1,40 @@
+package agent
+
+func updateConfigForECS(config map[string]interface{}, infraPlatform InfraPlatform) (map[string]interface{}, error) {
+
+	receiverData, ok := config["receivers"].(map[string]interface{})
```

Comment:
Define `receivers`, `processors` and other string literals as contants in `definitions.go` and use them here.

Reply:
created definitions for strings